src/rgw/driver/d4n/rgw_sal_d4n.h

```diff
@@ -139,7 +139,7 @@ class D4NFilterObject : public FilterObject {
                const DoutPrefixProvider* dpp, optional_yield y) override;
     virtual const std::string &get_name() const override { return next->get_name(); }
     virtual int set_obj_attrs(const DoutPrefixProvider* dpp, Attrs* setattrs,
-                            Attrs* delattrs, optional_yield y) override;
+                            Attrs* delattrs, optional_yield y, uint32_t flags) override;
```

Comment:
This change seems to be involved with a build error preventing it from going through a QA phase. 

/home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos9/DIST/centos9/MACHINE_SIZE/gigantic/release/19.2.1-431-gc78855bb/rpm/el9/BUILD/ceph-19.2.1-431-gc78855bb/src/test/rgw/test_d4n_filter.cc:690:53: error: no matching function for call to ‘rgw::sal::Object::set_obj_attrs(const DoutPrefixProvider*&, std::map<std::__cxx11::basic_string<char>, ceph::buffer::v15_2_0::list>*, NULL, const optional_yield::empty_t&)’

more details here: https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos9,DIST=centos9,MACHINE_SIZE=gigantic/88761//consoleFull